HealthKitReporter/Classes/Model/Payload/Quantity.swift

```diff
@@ -54,6 +55,7 @@ public struct Quantity: Identifiable, Sample {
     }
 
     init(quantitySample: HKQuantitySample, unit: HKUnit) throws {
+        self.uuid = quantitySample.uuid
```

Comment:
After it will be string, please use:
```suggestion
        self.uuid = quantitySample.uuid.uuidString
```